automation/Commands/browser_commands.py

```diff
@@ -133,7 +133,7 @@ def get_website(url, sleep, visit_id, webdriver,
         alert = webdriver.switch_to_alert()
         alert.dismiss()
         time.sleep(1)
-    except TimeoutException:
+    except (TimeoutException, WebDriverException):
```

Comment:
This will completely suppress the exceptions that are reported in https://github.com/mozilla/OpenWPM/issues/404 and prevent being able to properly address these errors as per https://github.com/mozilla/OpenWPM/issues/404#issuecomment-517879065. When addressing #404, we must remember to change this back to throwing exceptions, so that we can reproduce the root cause via tests, report upstream etc.